src/app/application/components/data-collection/DataCollection.vue

```diff
@@ -140,9 +144,6 @@ const paginated = computed(() => {
 
 </script>
 <style lang="scss" scoped>
-.pagination {
-  margin-top: $kui-space-70;
-}
```

Comment:
This is the value I was talking about in the PR description. We know let XLayout dictate the margin between the table and pagination (and it will be the same as all the other places we use XLayout)